app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -255,7 +255,7 @@ private static boolean updateDislike(AtomicReference<Object> textRef, boolean is
             }
             replacementSpannable = newSpannableWithDislikes(oldSpannable, voteData);
         } else {
-            String leftSegmentedSeparatorString = ReVancedUtils.isRightToLeftTextLayout() ? "\u200F|  " : "|  ";
+            String leftSegmentedSeparatorString = ReVancedUtils.isRightToLeftTextLayout() ? "\u200F|   " : "|   ";
```

Comment:
Why has a space been added?

Reply:
3 spaces looks a little better on the devices I used.  Although 2 looks ok as well I guess.  I'll change it back to 2 spaces.

Reply:
You can change the width of a span that contains the space and append it.